api/src/paths/dwc/eml.test.ts

```diff
@@ -0,0 +1,662 @@
+import chai, { expect } from 'chai';
+import { describe } from 'mocha';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import { getMockDBConnection } from '../../__mocks__/db';
+import * as db from '../../database/db';
+import { CustomError } from '../../errors/CustomError';
+import * as eml from './eml';
+import * as eml_queries from '../../queries/dwc/dwc-queries';
+import SQL from 'sql-template-strings';
+
+chai.use(sinonChai);
+
+const dbConnectionObj = getMockDBConnection({
+  systemUserId: () => {
+    return 20;
+  }
+});
+
+describe('getSurveyDataPackageEML', () => {
+  const sampleReq = {
+    keycloak_token: {},
+    body: {
+      data_package_id: 1
+    }
+  } as any;
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  it('should throw a 400 error when no data package id is provided', async () => {
+    sinon.stub(db, 'getDBConnection').returns(dbConnectionObj);
+
+    try {
+      const result = eml.getSurveyDataPackageEML();
+      await result(
```

Comment:
```suggestion
      const requestHandler = eml.getSurveyDataPackageEML();
      await requestHandler(
```
```suggestion
      const result = eml.getSurveyDataPackageEML();
      await result(
```

Reply:
(This isn't something you could have really known, as there are tons of cases of former style in the code base)

Calling the return value from `getSurveyDataPackageEML` (or any endpoint function) `result` is a bit misleading, as its really just a function (specifically a RequestHandler function).  Moving forward it would be a slight improvement to readability if we called them `requestHandler`.